### ISSUE TITLE
I want to just open an image! by show() - show is used to paste an image

### ISSUE THREAD

**Issue by shekaryf** [devforth/painterro#80]
Here is my code:
   var painterro = null;
    function my_show(image_url) {
        if (painterro != null) {
            painterro.show(image_url);
            return;
        }
        painterro = Painterro({
    ...
   **how_to_paste_actions: ['replace_all', 'paste_over'],**
   }
At first time that my "painterro" is null ,there is not any question dialog. But for second one it asks "How do you want to paste ?". 
I want to just open it by "show()" and when I click on "open" on menu, see this question dialog for paste.

_Actually, you have used the "show" function for showing and pasting of a picture,I want a function to just show whiteout any question for paste._

Is it possible?

Thanks.

**Comment by ivictbor** [comment]
Sorry, did not understand. Please rewrite it in format - current behaviour, expected behaviour

**Comment by shekaryf** [comment]
I changed my explanation. Any Idea?

**Comment by ivictbor** [comment]
As suggested in docs if you will leave one action in htpa you should never see dialog e.g. `how_to_paste_actions: ['replace_all']`. But if I understood right you want something more complex - you want to leave it for scenarios like Ctrl+V, drag-n-drop, open, but don't show it when you call `show` API.

Thanks for question rewrite, but some points are still look strange, like `At first time that my "painterro" is null ,there is not any question dialog`. If painterro is null there might be no dialog at all because painterro not created. What you tried to say here, I think, that first show call works without htpa dialog and subsequent calls show htpa dialog. To get rid of it please check source for htpa dialog. I think it should not show up if image is empty, so some kind of clearing should be enough before calling sequent show in your code.

**Comment by shekaryf** [comment]
Yes ,it is true. It is clear that there wouldn't be any  continuing  by a null object. You took my intention.
To be more clear I attacked two images; here is first time than I use "show". As you see ,there isn't any over dialog for pasting.
![first_time](https://user-images.githubusercontent.com/59958635/72873964-3c5c4700-3d06-11ea-8e0a-cc28cf8083de.png)
But at second time that I've used "show", a pasting dialog is shown.
![second_time](https://user-images.githubusercontent.com/59958635/72873963-3c5c4700-3d06-11ea-9553-b2ab90155c4d.png)
I want to do not see this dialog for second.
